docs/pages/blog/material-ui-is-now-mui.md

```diff
@@ -0,0 +1,108 @@
+At the same time, 5X more developers were struggling to customize the components.
+
+It was based on this context that we started work on v5 in 2019.
+Our primary focus was to revamp the **customization DX**.
+It had become clear that design (aesthetic, UX) and DX were key to unlock the next stage of growth.
+
+The new brand supports the v5.0.0 release while also creating space for new initiatives that broaden the company horizon. It's a big deal!
+
+Our ultimate goal is to become the most effective and efficient tool to build UIs, while making it accessible to the many.
+
+## Our new brand
+
+Let's dig in.
+
+### A new name
+
+We are breaking the strong association with Material Design
+as we have seen too many people confusing Material-UI with Google, or as a synonym of Material Design.
+
+We are now called **MUI**. It stands for **M**aterial to build **UI**s
+and is pronounced [/ɛm juː aɪ/](http://ipa-reader.xyz/?text=%C9%9Bmju%CB%90a%C9%AA).
+
+It's shorter, it distinguishes us, and it's familiar – many people already used it for abbreviating Material-UI.
+More importantly, it allows for the release of products not directly coupled to Material Design, such as an unstyled/headless version of the components, a brand new second design system as an alternative to MD, and more.
+
+### A new domain name
+
+A new name wouldn't be so official without a new domain,
+so we have moved home from https://material-ui.com/ to https://mui.com/, a very easy to type (and find) domain.
+
+### New logos
+
+We've tightened up the company logo to match the new brand.
+
+It's basically the same, to keep it familiar –
+we're keeping the geometrical shape, to resonate with the _building blocks_ idea of the components –
+however we're reducing the emphasis on the 3D perspective, stepping away from the notion of elevation that Material Design coined.
+
+<img loading="lazy" src="/static/blog/material-ui-is-now-mui/old-new.png" alt="" style="width: 100%; margin-bottom: 16px;" />
+
+To better identify, market, and promote each product, we're introducing logos for each one of them.
+
+<img loading="lazy" src="/static/blog/material-ui-is-now-mui/products.png" alt="" style="width: 100%; margin-bottom: 16px;" />
+
+### New package names
+
+The package names have changed in v5, which is a **breaking change**. You can find more details in the [release notes](https://github.com/mui-org/material-ui/releases/tag/v5.0.0).
+The npm organization name (scope) has moved from @material-ui to [@mui](https://www.npmjs.com/org/mui).
+
+### A new website & documentation
+
+Lastly, to celebrate the start of a new chapter for the company with the release of v5.0.0, and new products in the making, we're introducing a brand new website and documentation design.
+
+You'll find a completely different theme from Material Design – new typeface, colors, box-shadows, and more,
+all done using the sophisticated theming features of v5.
```

Comment:
```suggestion
all done using the powerfull theming features of v5.
```

```suggestion
all done using the advanced theming features of v5.
```

*sophisticated* feels like we are bragging :laughing:

Reply:
```suggestion
all done using the flexible theming features of v5.
```

Reply:
`flexible` works fine for me